ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/AuthHandlers.java

```diff
@@ -120,11 +139,18 @@ private static Handler<RoutingContext> wrappedHandler(
    * In order of preference, we see if the user is the system user, then we check for
```

Comment:
Can you modify this comment for the order of preference? Btw, wouldn't check for SYSTEM be the step 1, then check for SKIP path?